### ISSUE TITLE
Add configs for ACES 2.0

### ISSUE THREAD

**Issue by doug-walker** [AcademySoftwareFoundation/OpenColorIO-Config-ACES#128]
The configs need to be updated to support the release of ACES 2.0. There are many questions to resolve on the way to that goal, this issue is a placeholder for the discussion.

Open questions:
- What displays will be supported?  Waiting on a list from AMPAS.
- How will backwards compatibility for ACES 1 output transforms be handled? Will ACES 1 View Transforms and Views be included in the new configs, perhaps as inactive? Will the ACES 1 configs get further updates (e.g. AppleLog and bug fixes)?
- Which configs do we want built into the OCIO library itself?
- How will D60 sim be handled? All displays will have D65 and D60 versions. Is there a way for a user to make that decision at a project level and then only see the chosen white point? Do we need to add library support to allow hierarchical view menus (i.e. a family attribute for views)?
- And, I hesitate to even mention this, what should the new views be named?

**Comment by KelSolaar** [comment]
> How will backwards compatibility for ACES 1 output transforms be handled? Will ACES 1 View Transforms and Views be included in the new configs, perhaps as inactive? Will the ACES 1 configs get further updates (e.g. AppleLog and bug fixes)?

I don't think we should keep compatibility in the repo with ACES 1: It is going to be a huge mess. We will have to track two submodules, i.e., `aces-dev` and `aces-system`, we will have to maintain two different CTL discovery and parsing processes, we will have to maintain a giant spreadsheet with both ACEStransformID sets.
I would also prefer to keep the ACES 2 configs that we release devoid of ACES 1 things that are ported over and rather make sure that the OCIO config merging system works and delegate the frankensteining of configs to users.

**Comment by doug-walker** [comment]
Closing this as complete!  The bulk of the implementation and discussion is in PR #130.

The ACES 2.0 configs [are available here](https://github.com/AcademySoftwareFoundation/OpenColorIO-Config-ACES/releases).
